application/controllers/Views.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Views extends Application
+{
+
+	/**
+	 * Index Page for this controller.
+	 *
+	 * Maps to the following URL
+	 * 		http://example.com/
+	 * 	- or -
+	 * 		http://example.com/welcome/index
+	 *
+	 * So any other public methods not prefixed with an underscore will
+	 * map to /welcome/<method_name>
+	 * @see https://codeigniter.com/user_guide/general/urls.html
+	 */
+	public function index()
+	{
+        $this->data['pagetitle'] = 'Ordered TODO List';
+        $tasks = $this->tasks->all();   // get all the tasks
+        $this->data['content'] = 'Ok'; // so we don't need pagebody
+        $this->data['leftside'] = $this->makePrioritizedPanel($tasks);
+        $this->data['rightside'] = $this->tasks->makeCategorizedPanel($tasks);
+
+        $this->render('template_secondary');
+    }
+
+    public function makePrioritizedPanel($tasks) {
```

Comment:
again Allman notation here

Reply:
Have you ever thought maybe thats sexist?!
I refuse to participate in your patriarchy